andela/Ah-backend-xmen
# #162725244 Upgrade Django version from 1.0 to 2.1.5

#### What does this PR do?
This PR is to upgrade the Django framework from 1.0 to 2.1.5

#### Description of Task to be completed?
Upgrade the Django framework version to fix the application's security issues and also to have the application have the full support of python 3.5 and above

#### How should this be manually tested?
- fetch this branch by running the command `git fetch origin ch-upgrade-django-162725244` in the terminal.
- check into the branch `git checkout ch-upgrade-django-162725244`.
- start the virtual environment `source venv/bin/activate`
- re-install requirements `pip install -r requirements.txt` 
- then make migrations by running `python manage.py makemigrations`
- migrate by running `python manage.py migrate`
- start the application `python manage.py runserver`

#### What are the relevant pivotal tracker stories?
[#162725244](https://www.pivotaltracker.com/story/show/162725244)
